integration_testing/features/super-admin/super-admin-profile-command.feature

```diff
@@ -8,13 +8,15 @@ Feature: Super Admin profile manage command
   Scenario: A terminal is open in the same machine where Kolibri is running
```

Comment:
"Scenario name" should be more specific, even in cases when there is only one per `feature` file. How about something along the lines of `Execute the profile command and review the created CSV files`? 

Or separate actions in 2 or more scenarios: one to execute the command and check if it actually created the `_performance.csv`  file; second one to check if the file `_requests_performance.csv` file was created after some interactions; third one to open the files in text/spreadsheet editor and check if the recorded information is correct.

For a good example of small(er) steps described in separate scenarios in one feature file, check this:
https://github.com/learningequality/kolibri/blob/develop/integration_testing/features/coach/coach-lesson-resources-search.feature